backend/src/test-utils/server.ts

```diff
@@ -15,7 +16,13 @@ const initialiseServer = (session?: boolean): express.Application => {
   ;(app as any).authService = InitAuthService(redisService)
   ;(app as any).credentialService = InitCredentialService(redisService)
   sessionLoader({ app })
-  app.use(express.json())
+  app.use(
+    express.json({
+      // this must be bigger than transactionalEmail.bodySizeLimit so that users who exceed limit
+      // will get 404 error informing them of the size of the limit, instead of 500 error
+      limit: config.get('transactionalEmail.bodySizeLimit') * 10,
+    })
+  )
   app.use(express.urlencoded({ extended: false }))
```

Comment:
oh ya you're right — I will add that and the test cases (not sure whether `supertest` can do URLencoded calls, will look into that)

Reply:
OK fixed (similar behavior with the JSON payload — the tests give 413 error, but the actual error the user gets is 500 — weird)

Reply:
Wait where did the 413 come from? How come it's suddenly in the picture? We should be consistent with the status code that we're giving for this same error

Reply:
Honestly idk where it comes from. 
- On my API client, I only get the 500 error. This is the actual error code for the user. 
- I thought it might be from parts of the codebase where we set the 413 error code, but I changed those (to, e.g. use 400), but that has no effect on this test's behavior

So I think it's OK to return a 500 error code in this case, because the user is far exceeding our stated limits and, if they keep doing this, I don't think there is any problem with catching this weird behavior via our alerts.